templates/boot-fastboot/generic-fastboot-boot-template.jinja2

```diff
@@ -0,0 +1,93 @@
+{%- extends 'base/kernel-ci-base.jinja2' %}
+{% block metadata %}
+{{ super() }}
+{% endblock %}
+{% block main %}
+{{ super() }}
+protocols:
+  lava-lxc:
+    name: lxc-{{ platform }}-kernelci-fastboot
+    template: debian
+    distribution: debian
+    release: buster
+{% endblock %}
+{% block actions %}
+actions:
+{%- block deploy_lxc %}
+
+- deploy:
+    namespace: host
+    timeout:
+      minutes: 15
+    to: lxc
+    packages:
+    - android-tools-adb
+    - android-tools-fastboot
+    - mkbootimg
+    - cpio
+{%- endblock %}
+{%- block boot_lxc %}
+
+- boot:
+    namespace: host
+    prompts:
+    - 'root@(.*):/#'
+    timeout:
+      minutes: 5
+    method: lxc
+{%- endblock %}
+{%- block deploy_url %}
+
+- deploy:
+    timeout:
+      minutes: 40
```

Comment:
You're setting a timeout here that is larger than the job-wide default (10m).  This causes LAVA to complain loudly, and repeatedly that `Action timeout for <action> exceeds Job timeout`

Reply:
Presumably this is now resolved with the job timeout parameter?

Reply:
yes it's resolved.